lib/createjs.js

```diff
@@ -34,14 +52,7 @@ const createjsClb = (srcLng, trgLng, srcKeys, trgKeys, ntKeys, ns, cb) => {
     js.resources[ns] = {}
 
     Object.keys(srcKeys[ns]).forEach((srcKey) => {
-      js.resources[ns][srcKey] = {
-        source: srcKeys[ns][srcKey] || '',
-        target: trgKeys[ns][srcKey] || ''
-      }
-
-      if (ntKeys && ntKeys[ns] && ntKeys[ns][srcKey]) {
-        js.resources[ns][srcKey].note = ntKeys[ns][srcKey]
-      }
+      setSegment(js.resources[ns], srcKeys, trgKeys, ntKeys, srcKey)
```

Comment:
this should probably be something like: `setSegment(js.resources[ns], srcKeys[ns], trgKeys[ns], ntKeys && ntKeys[ns], srcKey)`

Reply:
@marmarosi this is for sure one of the test fails

Reply:
You're right, all tests succeeded having applied your patch. I just need to add some new tests for the changes.